src/pudl/transform/ferc1.py

```diff
@@ -3359,26 +3444,14 @@ def transform(
     """Make the module runnable for iterative testing during development."""
 
     ferc1_settings = Ferc1Settings(
-        # Do one year of each, type of data, or all the years:
+        # 1 year DBF + 1 year XBRL:
         years=[2020, 2021],
+        # All the years (slow with plants_steam_ferc1 included)
         # years=Ferc1Settings().years,
-        tables=[
-            "fuel_ferc1",
-            "plants_steam_ferc1",
-            "plants_hydro_ferc1",
-            "plant_in_service_ferc1",
-            "plants_pumped_storage_ferc1",
-            "purchased_power_ferc1",
-            "plants_small_ferc1",
-            "transmission_ferc1",
-            "electric_energy_sources_ferc1",
-            "electric_energy_dispositions_ferc1",
-            "utility_plant_summary_ferc1",
-            "balance_sheet_assets_ferc1",
-            "income_statement_ferc1",
-            "depreciation_amortization_summary_ferc1",
-            "retained_earnings_ferc1",
-        ],
+        # Just test your new table:
+        # tables=["your_new_table_ferc1"],
+        # Run all the tables to make sure you haven't broken something!
+        tables=Ferc1Settings().tables,
```

Comment:
People aren't remembering to update this, and I don't know if that means they think they're testing their new code, but aren't, or just aren't using it. But this way it will default to running all the tables with 2 years of data, even without being updated.

Reply:
I have been running the full ferc etl like this in a notebook before running the full tests. regardless, this is a good idea bc it has been hard to remember all of the little places to update these things.